### PR TITLE
Updated and renamed relativistic profile

### PR DESCRIPTION
**Pull request by dangoh123**
Tidied up the relativistic emission profile, using UTF-8 formatting where appropriate. Added comments to functions, and added labels to plots. Renamed the file for clarity.

Fairly sure that the shape we have so far looks about right, however the scale of the temperature plot is very far away from what we are expecting. Not sure yet what the issue is, but we are looking at the non-relativistic version first to see if the issue persists and will work back from there.

### CODE REVIEW

**Comment by fjebaker** [comment]
Nice! This looks cool, I'll review it properly later today! :)

File: src/Rel_PT_Emission_Profile.jl [hunk 1]
```diff
@@ -0,0 +1,73 @@
+#imports
+using Plots
+
+#functions
+function r_isco(a_star, M)
+    """
+    From Reynolds (2020) (Eq2-4)
+    Calculates the radius of the inner most stable circular orbit of the a black hole,
+    with spin a_star, and mass M.
+    """
```
Comment (fjebaker) [inline src/Rel_PT_Emission_Profile.jl:10]: Docstrings in Julia go before the function, i.e.
```julia
"""
    function r_isco(a_star, M)

From Reynolds (2020) (Eq2-4)
Calculates the radius of the inner most stable circular orbit of the a black hole,
with spin a_star, and mass M.
"""
function r_isco(a_star, M)
    # ...
end
```

What's also pretty nice is the docstrings can be full markdown text, with LaTeX functions in them too -- check out [this code from GeodesicTracer.jl](https://github.com/astro-group-bristol/GeodesicTracer.jl/blob/0225548b90655cac2491024b6d1ea05d3100e11d/src/coordinates.jl#L2-L11), which renders into [this documentation](http://www.star.bris.ac.uk/fbaker/docs/GeodesicTracer.jl/first_order_integration/#GeodesicTracer.%CE%A3).

We'll set up auto-documentation later down the line! :)

File: src/Rel_PT_Emission_Profile.jl [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+function temp(diss)
+    temp = (diss/σ_SB)^(1/4)
+end
+
+#constants
+G = 6.67e-11
+c = 3e8
+L_☼ = 3.8e26
+M_☼ = 1.99e30
+σ_SB = 5.67e-8
```
Comment (fjebaker) [inline src/Rel_PT_Emission_Profile.jl:53]: Nice ;)

**Review by fjebaker — COMMENTED** [review]
LGTM, I'll create the plots and paste them into the PR so we can easily see what this code does!

**Comment by fjebaker** [comment]
`dissplot`:

dissplot

`tempplot`:
tempplot

**Comment by fjebaker** [comment]
I just cleaned up the commit history in this PR so it's easier to see the commits you've added!

Merging now :)

**Comment by dangoh123** [comment]
> I just cleaned up the commit history in this PR so it's easier to see the commits you've added!
> 
> Merging now :)

Cheers, looks good.

**Pull request merged**